universal-login-sdk/lib/core/utils/getTargetAddress.ts

```diff
@@ -1,13 +1,13 @@
 import UniversalLoginSDK from '../..';
-import {isProperAddress, ensure} from '@universal-login/commons';
+import {isProperAddress, ensureNotNull} from '@universal-login/commons';
 import {InvalidAddressOrEnsName} from './errors';
 
 export const getTargetAddress = async (sdk: UniversalLoginSDK, addressOrEnsName: string) => {
   if (isProperAddress(addressOrEnsName)) {
     return addressOrEnsName;
   } else {
     const address = await sdk.resolveName(addressOrEnsName);
-    ensure(address, InvalidAddressOrEnsName, addressOrEnsName);
+    ensureNotNull(address, InvalidAddressOrEnsName, addressOrEnsName);
```

Comment:
I see a lot of duplication. 
I propose to move `ensureNotNull` to await `sdk.resolveName(...)`